src/ZprimeSelectionModule.cxx

```diff
@@ -141,7 +144,13 @@ bool ZprimeSelectionModule::process(Event & event){
   jetlepton_cleaner->process(event);
   jetER_smearer->process(event);
   jet_cleaner1->process(event); // jets w/ pt>25 GeV for lepton-2Dcut
-  bool pass_twodcut = twodcut_sel->passes(event); // lepton 2D-cut
+  //Trigger selection
+  bool ele_Trigger_selection = ele_Trigger->passes(event);
+  bool mu_Trigger_selection = mu_Trigger->passes(event);
+  if (!ele_Trigger_selection || !mu_Trigger_selection || (mu_Trigger_selection && ele_Trigger_selection)) return false;
```

Comment:
Hi Jochen, not sure what you mean by "trigger config". as far as i understand, one only has the "selection.xml" file that runs over the sample, and there you define several variables once-and-for-all, like the input files, the file branches and, i would say, the trigger that you want to use. if you think it's better to hard-code it no problem, i'm just not sure i got your point.

as for the two variables, to me it looked just unnecessary to define two variables when you know you re always going to use only one at runtime (one could, for example, use one variable and initialize it differently based on the 'channel' selection). i meant that more in terms of clarity of the code rather than its performance